tests/DbStorage_1.py

```diff
@@ -86,7 +86,7 @@ def testWriteRead(self):
 
         db.query()
 
-        self.assertTrue(next(db))
+        self.assertTrue(db.next())
```

Comment:
If `db` was meant to be iterable in the Python sense then we should add the `__iter__` method to `DbStorage` rather than trying to document our way out of it. I'm not entirely sure why that's missing from the class as it is found in others. I don't know where I would add the method.